workers/unity/Packages/com.improbable.gdk.core/Worker/StandaloneWorkerConnector.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using UnityEngine;
+
+namespace Improbable.Gdk.Core
+{
+    public abstract class StandaloneWorkerConnector : WorkerConnector
```

Comment:
Is it called standalone? What kind of worker isn't standalone?

Reply:
That's Unity terminology for building for OSX/Linux/Windows. Happy to rename it to something different, if you got a suggestion

Reply:
we could also call it `DefaultWorkerConnector`?

Reply:
I prefer `StandaloneWorkerConnector` to `DefaultWorkerConnector` as default can mean a lot of things. Also adhering to Unity's terminology makes sense in general to me.

Reply:
I think standalone refers to it being an executable. So for a build target that makes sense. For something connecting the, now running, worker it makes less sense.

Reply:
PCWorkerConnector or something maybe. It's also specifically for windows, mac, and linux too, not for the other standalone stuff like ps4

Reply:
Standalone also makes it sounds like an adjective on the worker connector. Basically I just don't like "standalone" as a word here.

Reply:
I'd also prefer Default to Standalone as that is actually more of what it is. Standalone is unity terminology but for something else really.